js/app/deviceListeners.js

```diff
@@ -20,22 +20,30 @@ define(['userData','weather', 'time', 'session'],function(userData, weather, tim
 		document.getElementById("profilePage").style.visibility = "hidden";
 	}
 
-	function screenOnOffListener() {
+	function screenOn() {

```

Comment:
No, updating time happens every second in gui.js. No need to do that also on a screenOn event, although the time will be updated a little faster if it's also done on a screenOn. So that might be an argument to do that also on a screenOn event.